src/tensor_ops/conv2d/mod.rs

```diff
@@ -88,6 +90,18 @@ where
     Const<{ (D + 2 * P - K) / S + 1 }>: Sized,
 {
     type Convolved = Const<{ (D + 2 * P - K) / S + 1 }>;
+
+    fn convolve_dim(&self) -> Self::Convolved {
+        Self::Convolved::default()
+    }
+}
+
+impl<const K: usize, const S: usize, const P: usize> ConvAlgebra<K, S, P> for usize {
+    type Convolved = usize;
+
+    fn convolve_dim(&self) -> Self::Convolved {
+        (self.size() + 2 * P - K) / S + 1
```

Comment:
```suggestion
        (self.size() + 2 * P).checked_sub(K).unwrap() / S + 1
```

I think we'll need to add some sort of check/assertion for when this conv operation is valid. The const generic version does this implicitly with the where bound